lib/shopify_transporter/pipeline/magento/order/addresses_attribute.rb

```diff
@@ -43,9 +59,7 @@ def address_attributes(input, prefix)
           end
 
           def name(address_attrs)
-            if address_attrs['firstname'].present? && address_attrs['lastname'].present?
-              address_attrs['firstname'] + ' ' + address_attrs['lastname']
-            end
+            (address_attrs['firstname'].to_s + ' ' + address_attrs['lastname'].to_s).strip
```

Comment:
We should leave this the same. Ideally, we shouldn't construct this if we have only first_name or only last_name.